pickee_api/urls.py

```diff
@@ -1,32 +1,46 @@
 from django.urls import include, path
 from rest_framework import routers
+from rest_framework_extensions.routers import NestedRouterMixin
 
-from pickee_api.views import auth, profile, recommendation
+from pickee_api import api
 
 app_name = 'pickee_api'
 
-# Routers provide an easy way of automatically determining the URL conf.
+
+class DefaultNestedRouter(NestedRouterMixin, routers.DefaultRouter):
+    pass
```

Comment:
For my own education:
Is it like a return true statement? (the class DefaultNestedRouter seems to be combining rest routes with normal routes)

Reply:
It injects the `NestedRouterMixin` [Mixin](https://www.django-rest-framework.org/api-guide/generic-views/#mixins) in the `routers.DefaultRouter` class. Composition!

Reply:
I mean "pass" statement